products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,233 @@
+  then on this path `count` would be `10`)
+* `:LOYAL_TO` - connects `Character` node with `Allegiance` node representing an allegiance the character is
+  loyal to
+* `:DIED_IN` and `:KILLED_IN` - connects `Character` node with `Death` node in which death happened 
+* `:HAPPENED_IN` - connects node `Death` with `Episode`, `Season` and `Location` nodes representing details
+  of death
+* `:PART_OF` connects node `Episode` with `Season` node which episode is part of
+
+### Importing the Snapshot
+
+We have prepared a database snapshot for this example, so the user can easily
+import it when starting Memgraph using the `--data-directory` option.
+
+```plaintext
+sudo -u memgraph \
+  /usr/lib/memgraph/memgraph --data-directory /usr/share/memgraph/examples/got \
+    --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+    --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+When using Memgraph installed from DEB or RPM package, the currently running
+Memgraph server may need to be stopped before importing the example. The user
+can do so using the following command:
+
+```plaintext
+systemctl stop memgraph
+```
+
+When using Docker, the example can be imported with the following command:
+
+```plaintext
+docker run -p 7687:7687 \
+  -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph \
+  memgraph --data-directory /usr/share/memgraph/examples/got \
+  --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+  --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+The user should note that any modifications of the database state will persist
+only during this run of Memgraph.
+
+### Example Queries using OpenCypher
+
+In the queries below, we are, as usual, using [OpenCypher](https://www.opencypher.org)
+to query Memgraph via the console.
+
+Here are some queries you might find interesting:
+
+**MINI-GAME** - If you have watched the TV Show, try to guess each result before executing the query!
+1) Let's start with a couple of simple queries. List the locations where most deaths occurred.
+Can you guess which one is it?
+
+```opencypher
+MATCH (l:Location)<-[:HAPPENED_IN]-(d:Death) 
+RETURN l.name AS location_name, COUNT(d) AS death_count
+ORDER BY death_count DESC
+```
+
+2) Now that we have the location with the most deaths, let's list the episodes with the most deaths
+as well.
+
+```opencypher
+MATCH (d:Death)-[:HAPPENED_IN]->(e:Episode)
+RETURN e.name AS episode_name, COUNT(d) AS kill_count
+ORDER BY kill_count DESC
+```
+
+3) List the number of kills per season. If you have watched the show, you should guess this one.
+
+```opencypher
+MATCH (d:Death)-[:HAPPENED_IN]->(s:Season) 
+RETURN s.number AS season_number, COUNT(d) AS death_count
+ORDER BY season_number ASC
+```
+
+4) Worst season by far, based on user ratings, is the last one, but can you guess the best one?

```

Comment:
_Worst_ => _The worst ..._
Also, here we have that opinionation again. Maybe consider using something along the lines of: `The most poorly rated season by far was the last one, ...`